Broken CI

So far seems like the CI is breaking as we require `node:crypto` ([Full log](https://github.com/expressjs/cookie-session/actions/runs/15467313049/job/43542359357)):

```
> cookie-session@2.1.0 test-ci /home/runner/work/cookie-session/cookie-session
> nyc --reporter=lcov --reporter=text npm test


> cookie-session@2.1.0 test /home/runner/work/cookie-session/cookie-session
> mocha --check-leaks --reporter spec --bail test/


Error: Cannot find module 'node:crypto'
Require stack:
- /home/runner/work/cookie-session/cookie-session/node_modules/@noble/hashes/cryptoNode.js
- /home/runner/work/cookie-session/cookie-session/node_modules/@noble/hashes/utils.js
- /home/runner/work/cookie-session/cookie-session/node_modules/@noble/hashes/sha3.js
- /home/runner/work/cookie-session/cookie-session/node_modules/@paralleldrive/cuid2/src/index.js
- /home/runner/work/cookie-session/cookie-session/node_modules/@paralleldrive/cuid2/index.js
- /home/runner/work/cookie-session/cookie-session/node_modules/formidable/src/Formidable.js
- /home/runner/work/cookie-session/cookie-session/node_modules/formidable/src/index.js
- /home/runner/work/cookie-session/cookie-session/node_modules/superagent/lib/node/index.js
- /home/runner/work/cookie-session/cookie-session/node_modules/supertest/lib/test.js
- /home/runner/work/cookie-session/cookie-session/node_modules/supertest/index.js
- /home/runner/work/cookie-session/cookie-session/test/test.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:962:[15](https://github.com/expressjs/cookie-session/actions/runs/15467325733/job/43543105402?pr=186#step:10:16))
```

Help wanted to investigate it!

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -79,33 +79,35 @@ jobs:
 
         - name: Node.js 8.x
           node-version: "8.17"
-          npm-i: mocha@7.2.0 nyc@14.1.1
+          npm-i: mocha@7.2.0 nyc@14.1.1 supertest@6.1.6
 
         - name: Node.js 9.x
           node-version: "9.11"
-          npm-i: mocha@7.2.0 nyc@14.1.1
+          npm-i: mocha@7.2.0 nyc@14.1.1 supertest@6.1.6
 
         - name: Node.js 10.x
           node-version: "10.24"
-          npm-i: mocha@8.4.0
+          npm-i: mocha@8.4.0 supertest@6.1.6
 
         - name: Node.js 11.x
           node-version: "11.15"
-          npm-i: mocha@8.4.0
+          npm-i: mocha@8.4.0 supertest@6.1.6
 
         - name: Node.js 12.x
           node-version: "12.22"
-          npm-i: mocha@9.2.2
+          npm-i: mocha@9.2.2 supertest@6.1.6
 
         - name: Node.js 13.x
           node-version: "13.14"
-          npm-i: mocha@9.2.2
+          npm-i: mocha@9.2.2 supertest@6.1.6
 
         - name: Node.js 14.x
           node-version: "14.21"
+          npm-i: mocha@9.2.2 supertest@6.1.6
 
         - name: Node.js 15.x
           node-version: "15.14"
+          npm-i: mocha@9.2.2 supertest@6.1.6
 
         - name: Node.js 16.x
           node-version: "16.20"

